native/account.go

```diff
@@ -0,0 +1,14 @@
+package native
+
+import (
+	"github.com/block-vision/sui-go-sdk/signer"
+)
+
+// CreateSigner creates a signer
+func CreateSigner(mnemonic string) (*signer.Signer, error) {
```

Comment:
this is a Sui signer, not a native signer. Should be in a `sui` package.

Reply:
done